### PR TITLE
Improve readability

### PR DESCRIPTION
**Pull request by hassankhan**
## What did you implement:

Improved readability of docs for AWS API Gateway sharing between multiple Serverless projects.

## How did you implement it:

Rewrote parts of the docs.

## How can we verify it:

Read the docs.

## Todos:

- [ ] ~~Write tests~~
- [X] Write documentation
- [ ] ~~Fix linting errors~~
- [ ] ~~Make sure code coverage hasn't dropped~~
- [ ] ~~Provide verification config / commands / resources~~
- [X] Enable "Allow edits from maintainers" for this PR
- [X] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

**Comment by horike37** [comment]
Thank you for the improvement @hassankhan :+1: 

Hey @alexdebrie, could you review this PR? It is hard to judge it for me since I'm non-native English speaker.

File: docs/providers/aws/events/apigateway.md [hunk 1]
```diff
@@ -858,7 +858,7 @@ Now that you have these two CloudFormation templates defined in your `serverless
 
 ## Share API Gateway and API Resources
 
-As you application grows, you will have idea to break it out into multiple services. However, each serverless project generates new API Gateway by default. If you want to share same API Gateway for muliple projects, you 'll need to reference REST API ID and Root Resource ID into serverless.yml files
+As your application grows, you will likely need to break it out into multiple, smaller services. By default, each Serverless project generates a new API Gateway by default. However, you can share the same API Gateway between multiple projects by referencing its' REST API ID and Root Resource ID in `serverless.yml` as follows:
```
Comment (alexdebrie) [inline docs/providers/aws/events/apigateway.md]: In the second sentence, could you remove either the opening "By default, " or the ending "by default" ?

In the third sentence, remove the apostrophe from `its'`.

Comment (hassankhan) [reply docs/providers/aws/events/apigateway.md]: #redundantredundancyerror 😄

**Comment by horike37** [comment]
Cool! Thanks @hassankhan and @alexdebrie :smile:

**Comment by hassankhan** [comment]
Always a pleasure @horike37 😄

**Pull request merged**